release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -151,6 +151,12 @@ Packet data is not sent to the network flows collector. Packet-level metadata su
 
 For more information, see xref:../networking/ovn_kubernetes_network_provider/tracking-network-flows.adoc#tracking-network-flows[Tracking network flows].
 
+[id="ocp-4-8-network-policy-audit-logs"]
+==== Network policy audit logs
+
+If you use the OVN-Kubernetes cluster network provider, you can enable audit logging for network policies in a namespace. The logs are in a syslog compatible format and saved locally, but can also be sent over a UDP connection or directed to a UNIX domain socket. You can specify whether to log allowed connections, dropped connections, or allowed and dropped connections. For more information, see
```

Comment:
@jboxman Could this be rephrased "You can specify whether to log allowed connections, dropped connections, or allowed and dropped connections" to "You can specify whether to log allowed connections and/or dropped connections by the network policy". or "You can specify whether to log allowed connections, dropped connections, or both allowed and dropped connections". Just a nit. Thanks!